server/src/lib/db.ts

```diff
@@ -18,9 +18,14 @@ export default class DB {
   }
 
   createAll() {
-    return this.user.create()
-        .then(() =>{return this.badges.create()})
-        .then(() =>{return this.progress.create()})
+    return this.user
+      .create()
+      .then(() => {
+        return this.badges.create();
+      })
+      .then(() => {
+        return this.progress.create();
```

Comment:
minor: using short-form arrow functions (`() => this.progress.create()`, i.e. without block and `return`) would allow these to be more compact

Reply:
it seems Prettier expands all functions. i agree with the compactness, but don't mind the expansion either